scripts/requireDocker.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if ! command -v docker &> /dev/null
+then
+  echo "Docker is required. Please install Docker and make sure it is running, and then try again."
+  exit 1
+fi
```

Comment:
Add +x to the permissions for this file. Please make sure ALL scripts have this :-)

Reply:
done